manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Mathpix.MathpixSnippingTool
+PackageVersion: 03.00.0074
+PackageLocale: en-US
+Publisher: Mathpix
+PackageName: Mathpix Snipping Tool
+License: Copyright (c) Mathpix

```

Comment:
A copyright is not a license; Please update this to an appropriate license such as `Proprietary`. `GPL-2.0`, `MIT`, or whatever the actual software license is

Reply:
Found it on https://github.com/Mathpix/mathpix-markdown-it/blob/master/LICENSE. It's a MIT License.

Reply:
Simply cannot find the license. I've check nearly everything.
Does the software NEED a license?